server_addon/blender/client/ayon_blender/plugins/create/create_model.py

```diff
@@ -29,3 +30,11 @@ def create(
                 obj.parent = asset_group
 
         return asset_group
+
+    def get_instance_attr_defs(self):
+        return [
+            NumberDef("unitScale",
+                      label="Unit Scale (FBX)",
+                      default=1.0,
+                      tooltip="Scale of the model, valid only for FBX export.")
+        ]
```

Comment:
I wonder if these are better implemented as attribute definitions on the Extractors instead since it seems so tied to that exporter?

Reply:
It is either that we add the collector which is specific to collect the unit scale for fbx exporter. Adding the definition into the extractor seems to have the same sort of confusion with the definition being implemented here.